controls/srg_ctr/SRG-APP-000091-CTR-000160.yml

```diff
@@ -6,4 +6,19 @@ controls:
     attempts to access privileges occur.
   related_rules:
   - audit_profile_set
-  status: inherently met
+  rules:
+  - audit_rules_dac_modification_chmod
+  - audit_rules_dac_modification_chown
+  - audit_rules_dac_modification_fchmod
+  - audit_rules_dac_modification_fchmodat
+  - audit_rules_dac_modification_fchown
+  - audit_rules_dac_modification_fchownat
+  - audit_rules_dac_modification_fremovexattr
+  - audit_rules_dac_modification_fsetxattr
+  - audit_rules_dac_modification_lchown
+  - audit_rules_dac_modification_lremovexattr
+  - audit_rules_dac_modification_lsetxattr
+  - audit_rules_dac_modification_removexattr
+  - audit_rules_dac_modification_setxattr
+  - audit_ospp_general
```

Comment:
Nice catch, I agree that those are "nicer" than the full OSPP rule. I added e2e test stanzas, let's see how the CI run will do.
About the 32bit rules, I think it would be nice because there still might be customers who run a legacy 32bit app even though I think it's quite rare. I filed https://issues.redhat.com/browse/CMP-2079 to track this - I don't think it's a critical issue, but it's something worth fixing in the next update.